explorer/graphql/data/chains_provider.go

```diff
@@ -20,31 +20,36 @@ type ChainsProvider struct {
 func NewChainsProvider(network netconf.ID) *ChainsProvider {
 	// Define the chains supported by each network
 	networks := map[netconf.ID][]struct {
-		id      uint64
-		name    string
-		logoURL string
+		id          uint64
+		name        string
+		logoURL     string
+		addrUrlFmt  string
+		blockURLFmt string
+		txURLFmt    string
 	}{
 		netconf.Devnet: {
-			{id: uint64(evmchain.IDOmniEphemeral), name: "Omni Ephemeral", logoURL: "https://chainlist.org/unknown-logo.png"},
-			{id: uint64(evmchain.IDMockOp), name: "Mock Op", logoURL: "https://icons.llamao.fi/icons/chains/rsz_optimism.jpg"},
-			{id: uint64(evmchain.IDMockArb), name: "Mock Arb", logoURL: "https://icons.llamao.fi/icons/chains/rsz_arbitrum.jpg"},
+			{id: uint64(evmchain.IDOmniEphemeral), name: "Omni Ephemeral", logoURL: "https://chainlist.org/unknown-logo.png", addrUrlFmt: "", blockURLFmt: "", txURLFmt: ""},
+			{id: uint64(evmchain.IDMockL1Fast), name: "Mock L1 Fast", logoURL: "https://icons.llamao.fi/icons/chains/rsz_optimism.jpg", addrUrlFmt: "https://sepolia.arbiscan.io/address/%s", blockURLFmt: "https://sepolia.arbiscan.io/block/%d", txURLFmt: "https://sepolia.arbiscan.io/tx/%s"},
```

Comment:
I was not sure what URLs to put for Mock L1 Fast and Mock L2. Just put something for now to display in the unit tests. URLs will actually only make sense on Testnet and Mainent.